### PR TITLE
New pest control procedure

### PR DESCRIPTION
**Pull request by hx-markterry**
As discussed in the last meeting.

@jackdcrawford @lizpope @chandlax for review please

### CODE REVIEW

File: pest-control-process.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+A member of each pod is encouraged to attend to assist in the assignment of bugs and contribute to improving this process.
+
+* Chaired by developer on rotation.
+* Triage current bugs.
+* Assign bugs to pod backlogs.
+* What’s happened since last time?
+  * Monitor progress of bug completion.
+  * Recurring themes.
+* Review current process.
+* Sent email update.
```
Comment (lizpope) [inline pest-control-process.md]: typo - should be 'send' not 'sent'

File: pest-control-process.md [hunk 2]
```diff
@@ -0,0 +1,134 @@
+# Pest Control Process
+
+This is an important part of our developer role here. It allows us time to catch and resolve bugs and issues which we would not otherwise see or prioritised. This creates a better development environment for us all as well as improving the customers experience on our site.
```
Comment (lizpope) [inline pest-control-process.md]: see or prioritised?

don't need the or.

File: pest-control-process.md [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+## tl;dr
+- Check bug reporting systems regularly
+- One developer responsible on a daily basis
+- Group meetings once a sprint.
+
+## Process
+
+Expedited issues are not covered here, please see the expedited proceed for details.
+
+The most important part of Bugsnag is communication. You should be talking to people about what you're seeing and updating on things you're working on etc. Don't suffer in silence on a tough bug, ask for help. And don't assume an issue is being looked at, check!
```
Comment (jamesmcneil) [inline pest-control-process.md]: `Pest Control` instead of `Bugsnag` here?

File: pest-control-process.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+
+* Developer rotated on a weekly basis.
+  * Handled by SE's which are able to delegate as they see fit.
```
Comment (jamesmcneil) [inline pest-control-process.md]: No apostrophe in `SE's`?

File: pest-control-process.md [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+  * Monitor progress of bug completion.
+  * Recurring themes.
+* Review current process.
+* Send email update.
+
+## Systems used
+
+### Bugsnag
+
+You'll require our Bugsnag credentials. Ask the last person on bugsnag for these.
```
Comment (jamesmcneil) [inline pest-control-process.md]: `last person on Pest Control`?

File: pest-control-process.md [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+![](/images/pest-control-process/project-selector.png)
+
+Errors can be viewed in either a grouped or ungrouped state.
+
+![](/images/pest-control-process/grouping.png)
+
+Most people prefer the ungrouped view as the grouping isn't always accurate.
+
+##### Grouped View
+When viewing grouped errors you can however get an idea of the bugs importance.
```
Comment (jamesmcneil) [inline pest-control-process.md]: `bugs` should have an apostrophe `bug's importance`

File: pest-control-process.md [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+No one likes a buggy application, so why should we build one?
+
+## tl;dr
+- Check bug reporting systems regularly
+- One developer responsible on a daily basis
+- Group meetings once a sprint.
+
+## Process
+
+Expedited issues are not covered here, please see the expedited proceed for details.
```
Comment (jamesmcneil) [inline pest-control-process.md]: proceed? Should be `process` or `procedure`?

**Pull request merged**